test/examples/logical/logical_sos.jl

```diff
@@ -37,7 +37,7 @@ function test_logical_sos1()
         # :: Reformulation ::
         qubo_model = unsafe_backend(model)
```

Comment:
Why do you need the unsafe backend here still, yet in the other tests you don't?

Reply:
You're right, they are not needed anymore!